test/browser/wrapper.jsx

```diff
@@ -0,0 +1,181 @@
+/* global before, beforeEach, after, describe, expect, it */
+import Wrapper from '../../src/wrapper'
+
+const injectSelectToEnhanceIntoDOM = (element, id, name, options, selectedOption) => {
+  var $select = document.createElement('select')
+  if (id) {
+    $select.id = id
+  }
+  if (name) {
+    $select.name = name
+  }
+
+  for (var optionKey in options) {
```

Comment:
I don't think this is worth it, you just end up with duplicated code since DOM manipulations are not functional.